### PR TITLE
chore(dependabot): exclude react along with related packages from npm updates

### PR DESCRIPTION
**Pull request by wangerekaharun**
### **User description**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

 exclude react from npm updates

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

___

### **PR Type**
Other

___

### **Description**
• Exclude React from Dependabot npm updates
• Add React to ignore list alongside React Native
• Update comments for better clarity
• Apply changes to both root and example directories

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>dependabot.yml</strong><dd><code>Add React to Dependabot ignore list</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

.github/dependabot.yml

• Added <code>react</code> to the ignore list for npm updates in root directory<br> • <br>Added <code>react</code> to the ignore list for npm updates in example directory<br> • <br>Improved comments formatting and clarity<br> • Removed trailing whitespace <br>at end of file

</details>

  </td>
  <td><a href="https://github.com/smileidentity/react-native/pull/108/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28">+8/-4</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> <details> <summary>  Need help?</summary><li>Type <code>/help how to ...</code> in the comments thread for any questions about PR-Agent usage.</li><li>Check out the <a href="https://qodo-merge-docs.qodo.ai/usage-guide/">documentation</a> for more information.</li></details>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -16,7 +16,28 @@ updates:
...
+      - dependency-name: react-native/typescript-config
+      - dependency-name: react-native/eslint-config
+      - dependency-name: react-navigation/native
+      - dependency-name: react-navigation/native-stack
+      - dependency-name: react-native-safe-area-context
+      - dependency-name: react-native-screens
+      - dependency-name: types/react
+      - dependency-name: types/jest
+      - dependency-name: eslint
+      - dependency-name: prettier
```
Comment (jumaallan) [inline .github/dependabot.yml:33]: Even these need to be excluded?

Comment (wangerekaharun) [reply .github/dependabot.yml:33]: Let me remove theme
